plugin/trino-hive/src/main/java/io/trino/plugin/hive/parquet/MetricsUtils.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.hive.parquet;
+
+import io.trino.filesystem.InputFileMetrics;
+import io.trino.plugin.base.metrics.LongCount;
+import io.trino.spi.metrics.Metric;
+import io.trino.spi.metrics.Metrics;
+
+import java.util.Map;
+
+public final class MetricsUtils
+{
+    private MetricsUtils() {}
+
+    public static Metrics convert(InputFileMetrics inputMetrics)
```

Comment:
I'm not sure about this class - at first, I wanted to place `convert` method in `InputFileMetrics` but it's not aware of `LongCount`. Then, I thought of placing `MetricsUtils` at `trino-plugin-toolkit`, but it's not aware of `InputFileMetrics`, so I ended up placing it here.